sevsnpmeasure/vcpu_types.py

```diff
@@ -39,4 +41,5 @@ def cpu_sig(family: int, model: int, stepping: int) -> int:
     'EPYC-Milan': cpu_sig(family=25, model=1, stepping=1),
     'EPYC-Milan-v1': cpu_sig(family=25, model=1, stepping=1),
     'EPYC-Milan-v2': cpu_sig(family=25, model=1, stepping=1),
+    'ec2': cpu_sig(family=0, model=0, stepping=0),
```

Comment:
Will a client ever know exactly which CPU they are running on or does EC2 obscure that info in favor of a more generic offering? Would a client ever be concerned about not knowing the CPU details, for instance if some hardware defect were discovered?

Do you see all future SEV CPU types on EC2 having the same boot process or will we need a versioned EC2 instance type at some point?